Week3/homework/youAreAmazing.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+{
+    function giveCompliment(){
```

Comment:
The question asks for the function to take one argument, `name`. Please update :)